freqtrade/commands/cli_options.py

```diff
@@ -658,6 +658,21 @@ def __init__(self, *args, **kwargs):
         nargs='+',
         default=[],
     ),
+    "analysis_rejected": Arg(
+        '--rejected-signals',
+        help='Analyse rejected signals',
+        action='store_true',
+    ),
+    "analysis_to_csv": Arg(
+        '--analysis-to-csv',
+        help='Save selected analysis tables to individual CSVs',
+        action='store_true',
+    ),
+    "analysis_csv_path": Arg(
+        '--analysis-csv-path',
+        help=("Specify a path to save the analysis CSVs "
+              "if --analysis-to-csv is enabled. Default: user_data/basktesting_results/"),
+    ),
```

Comment:
Any reason we can't combine this with `export_csv` (see above in this file)? 
maybe there's good reasons to split it - maybe it's because you didn't notice ... ?

Reply:
I did notice, but it would require changing the description in the command, which I wasn't sure you'd prefer. I can easily rework this to use export_csv.

Reply:
well the example (hyperopt.csv) is dedicated to where it's currently used.
if it's used elsewhere, too - there's no reason to not change this to something more generic.

Reply:
Would you recommend using the `exportfilename` option for this also, like the `export_csv` above?

Reply:
well iirc - then `export_csv` is a path relative to CWD - so it would not need 2 options for this (you can just pass `userdata/backtests/whatever.csv`).
Might need checking though - as it's been ages since i touched the csv code